src/components/Landing/QAs.tsx

```diff
@@ -18,19 +19,19 @@ export const JBDiscordLink = ({ children }: PropsWithChildren<{}>) => (
 // Saw the suggestion to separate the store and render into 2 files here:
 // https://github.com/lingui/js-lingui/issues/707#issuecomment-657199843
 // Not sure why but this fixed the problem.
-// Take-away: If you're storing a list of <Trans>` strings in a list,
+// Take-away: If you're storing a list of <Trans key={uniqueId()}>` strings in a list,
 // make sure you're not rendering it from the same file.
 export default function QAs() {
   return [
     {
       q: <Trans>Who funds Juicebox projects?</Trans>,
       a: [
-        <Trans>
+        <Trans key={uniqueId()}>
```

Comment:
This I think will still suffer the same problem. Have a look in the article I linked, especially surrounding the example the author gives of `key={Math.random()}`

Definitely think you're on the right track though

Reply:
tbh though - I am wondering if it might be okay for Trans.... 🤔

Reply:
Seems kind of useless having the key for trans as is. Wonder if the docs have anything about bypassing the need for Trans key

Reply:
Yeah this a great point, thanks for calling it out.

I ended up refactoring this whole QA thing to avoid this problem entirely.